fatips/102.md

```diff
@@ -14,93 +14,66 @@ tokens.
 
 Create a central, authoritative, public index of FAT tokens. Such an index is
 ideal for creating FAT token explorers, and improving the discoverability of
-tokens and the ecosystem.
+tokens within the ecosystem.
 
 
 # Specification
 
 
-## Index Chain
+## Token Index Chain
 
-A single chain, the index chain holds all entries supporting the standard
+The index chain holds entries corresponding to individual signed listings of FAT tokens by their issuers.
```

Comment:
You are using the term `listing` which I personally think is much better than `indexing` used elsewhere in this document. When I read indexing I thought it was a mechanism or structure allowing queries of tokens by multiple criteria such as date of creation, symbol etc. I believe naming it a `listing` or `register` spec fit better what this really is about.

Reply:
Where do you propose the term "listing" to be used instead of indexing?. The data on the chain is mainly intended to help a program create an index of tokens that can be searched like you are mentioning, but the data on chain itself is not an index, just references.

Reply:
I'd remove indexing pretty everywhere from that doc. That document itself is called "Token Indexing Standard", so "Token Listing/Register Standard".
Indexing of tokens is one potential use case this standard, but it is not what this standard define in itself, it defines a way for tokens to register themselves publicly.

Reply:
I can make the actual change if you guys agree on the idea, no problem

Reply:
I think it's a better term, sorry I didn't mention that haha :rocket: Sure go ahead